ctrlf.el

```diff
@@ -35,6 +35,13 @@
   "Whether highlight current line or not."
   :type 'boolean)
 
+(defcustom ctrlf-translate-regexp nil
```

Comment:
Could we call this e.g. `ctrlf-split-search-string` so that its function is more readily apparent?

Actually, I think the better solution would be to generalize this option to specify an arbitrary transformation of the user input into a regexp, and then categorically use `re-search-forward|backward` as the internal interface. Then we have a tristate for a `ctrlf-input-style` option:

* `literal`
* `regexp`
* `fuzzy`
* `fuzzy-regexp` (although this one seems sketchy to me)
* things like `symbol`, etc.

Then, #10 can be solved in a more general way.

However, you don't have to implement all that now if you don't want to. It can be done later.

Reply:
About the input style thing: It's not the goal of this PR. Seems you are describing converting a literal input to a regexp, but I'm converting from a regexp (with that "spaces rule") to another regexp. It's just a little thing helping us do regexp search, so `ctrlf-split-search-string` doesn't describe it.